### DISCUSSION TITLE
board.mk file missing for the esp32s2 boards

### DISCUSSION THREAD

**Discussion by Ivo44** [hathach/tinyusb#964] [Q&A]
I'm trying to compile USB midi test for the feather esp32s2, and after fishing for the includes,

I get the error in "tusb_config.h" that CFG_TUSB_MCU must be defined.
Inside "tusb_config.h" it says it's defined it "board.mk", but such file doesn't exist for any of the esp32s2 boards, only a "board.cmake" and "board.h".

Am I missing something here? Or is the board.mk really not present for the esp32s2?

**Comment by perigoso** [comment]
The esp32 does not use regular make for its build system, hence why you can't find board.mk, board.cmake does its job.

if you look inside you will find
```
target_compile_options(${COMPONENT_TARGET} PUBLIC
  "-DCFG_TUSB_MCU=OPT_MCU_ESP32S2"
  "-DCFG_TUSB_OS=OPT_OS_FREERTOS"
)
```
that sets that particular variable.

not sure why you cant compile.
